### PR TITLE
[remoteopenhab] Catch exception when parsing item state

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/handler/RemoteopenhabBridgeHandler.java [hunk 1]
```diff
@@ -484,108 +484,108 @@ private void updateChannelState(String itemName, @Nullable String stateType, Str
...
+                        }
+                        break;
+                    default:
+                        logger.debug("Item type {} is not yet supported", acceptedItemType);
+                        break;
+                }
             }
+        } catch (IllegalArgumentException | DateTimeException e) {
+            logger.warn("Failed to parse state \"{}\" for item {}: {}", state, itemName, e.getMessage());
+            channelState = null;
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/handler/RemoteopenhabBridgeHandler.java:588]: Shouldn't it set the state to `UNDEF`? to indicate something is wrong. By setting it to null it seems to be used previously as a completely unsupported and thus nothing was updated (maybe in cases that could theoretically not happen)

Comment (lolodomo) [reply bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/handler/RemoteopenhabBridgeHandler.java:588]: Looks like a good idea.

Comment (lolodomo) [reply bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/handler/RemoteopenhabBridgeHandler.java]: Done

**Review by Hilbrand — APPROVED** [review]
LGTM

**Review by cpmeister — APPROVED** [review]

**Pull request merged**